src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,109 @@
+.sidebar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+}
+.sidebar .menu_item {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  border-top: 1px solid #1b64ab;
+  color: #fff;
+}
+.sidebar ul li:hover {
+  background-color: #00407a;
+}
+.sidebar .menu_footer {
+  color: #fff;
+  margin-top: auto;
+  padding-top: 12px;
+  padding-bottom: 12px;
+}
+.sidebar .menu_footer ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+.sidebar .menu_footer ul li {
+  display: flex;
+  align-items: left;
+  padding: 16px 24px;
+}
+.sidebar .text {
+  margin-left: 12px;
+  font-family: "Open Sans-Regular", Helvetica;
+  font-size: 1.2rem;
+  font-weight: 100;
+  letter-spacing: 1.94px;
+  line-height: 2.5rem;
+}
+.sidebar .icon svg path {
+  stroke: #fff;
+}
+.sidebar .menu_footer ul li .icon {
```

Comment:
Let's use a consistent class naming convention.  There is a mix of `-` and `_` for separating words in a name.  Let's stick with `-` for everything as it's more common for css classes.